vnext/Microsoft.ReactNative/Views/DevMenu.cpp

```diff
@@ -106,6 +111,24 @@ void DevMenuManager::CreateAndShowUI() noexcept {
       Mso::React::ReactOptions::UseFastRefresh(m_context->Properties()) ? L"Disable Fast Refresh"
                                                                         : L"Enable Fast Refresh");
 
+  devMenu.SamplingProfilerText().Text(
```

Comment:
Is the sampling profiler Hermes specific? If so, it should not show up (or at least it should be disabled) when using Chakra / V8, right?

Reply:
I remember you recently added the ability to switch to hermes at runtime but I forget which #define that is gated by. Does USE_HERMES mean "hermes is available" or "actually use hermes"?
I was under the impression that for something like this you would have to query the instance settings to see whether the JS engine is hermes or not, what am I missing?

Reply:
I saw we started gating this on preprocessor definition in a new commit, but think this will still have the backing XAML elements present.

Curious why we change code behavior at preprocessor time instead of runtime? I've always felt like #ifdefs make code harder to reason about. Is it possible to identify Hermes usage at runtime?

Reply:
Yeah.. It would be more correct to use the instance setting .. Let me see how to exopse it and make it available here.